src/main/java/org/wso2/securevault/secret/SecretManager.java

```diff
@@ -100,71 +132,50 @@ public void init(Properties properties) {
             }
         }
 
-        String repositoriesString = MiscellaneousUtil.getProperty(
-                configurationProperties, PROP_SECRET_REPOSITORIES, null);
-        if (repositoriesString == null || "".equals(repositoriesString)) {
-            if (log.isDebugEnabled()) {
-                log.debug("No secret repositories have been configured");
-            }
-            return;
-        }
+        loadProviders(configurationProperties);
 
-        String[] repositories = repositoriesString.split(",");
-        if (repositories == null || repositories.length == 0) {
-            if (log.isDebugEnabled()) {
-                log.debug("No secret repositories have been configured");
-            }
+        if (!isSecureVaultStatusValidated()) {
```

Comment:
shouldnt this be isSecureVaultStatusValid ?

Reply:
Changed